floodviz/map_utils_tests.py

```diff
@@ -0,0 +1,171 @@
+                            "{ \"type\": \"Feature\",\n"
+                            " \"geometry\": {\n"
+                            " \"type\": \"Point\",\n"
+                            " \"coordinates\" : [-92.2574003, 42.83609117]\n"
+                            " },\n"
+                            " \"properties\": {\n"
+                            " \"name\": \"Wapsipinicon River near Tripoli, IA\",\n"
+                            " \"id\": \"05420680\",\n"
+                            " \"huc\": \"07080102\" \n"
+                            " } \n"
+                            " },\n"
+                            "{ \"type\": \"Feature\",\n"
+                            " \"geometry\": {\n"
+                            " \"type\": \"Point\",\n"
+                            " \"coordinates\" : [-92.4632451, 42.4077639]\n"
+                            " },\n"
+                            " \"properties\": {\n"
+                            " \"name\": \"Black Hawk Creek at Hudson, IA\",\n"
+                            " \"id\": \"05463500\",\n"
+                            " \"huc\": \"07080205\" \n"
+                            " } \n"
+                            " }\n"
+                            " ] }")
+        self.missing_huc_return = ("{ \"type\": \"FeatureCollection\", \"features\": [ \n"
+                                   "{ \"type\": \"Feature\",\n"
+                                   " \"geometry\": {\n"
+                                   " \"type\": \"Point\",\n"
+                                   " \"coordinates\" : [-92.2574003, 42.83609117]\n"
+                                   " },\n"
+                                   " \"properties\": {\n"
+                                   " \"name\": \"Wapsipinicon River near Tripoli, IA\",\n"
+                                   " \"id\": \"05420680\",\n"
+                                   " \"huc\": \"07080102\" \n"
+                                   " } \n"
+                                   " },\n"
+                                   "{ \"type\": \"Feature\",\n"
+                                   " \"geometry\": {\n"
+                                   " \"type\": \"Point\",\n"
+                                   " \"coordinates\" : [-92.4632451, 42.4077639]\n"
+                                   " },\n"
+                                   " \"properties\": {\n"
+                                   " \"name\": \"Black Hawk Creek at Hudson, IA\",\n"
+                                   " \"id\": \"05463500\",\n"
+                                   " \"huc\": \"n/a\" \n"
+                                   " } \n"
+                                   " }\n"
+                                   " ] }")
+
+    def test_good_data(self):
+        self.assertEqual(get_sites_geojson(self.good_data), self.good_return)
+
+    def test_missing_huc(self):
+        self.assertEqual(get_sites_geojson(self.missing_huc_data), self.missing_huc_return)
+
+if __name__ == '__main__':
```

Comment:
This is not necessary when using nose to run your tests. The reason to use nose is it collects all of your tests for you. So you can run the tests for an entire application with one command.